css/index.css

```diff
@@ -1,3 +1,552 @@
+}
+
+header a:hover {
+  animation: headerBtnFade 0.4s forwards ease-out;
+}
+
+header a:active {
+  animation: headerBtnClick 0.2s forwards ease-out;
+}
+
+@keyframes headerBtnFade {
+  0% {
+    background-color: lightblue;
+    color: black;
+  }
+  100% {
+    background-color: black;
+    color: white;
+  }
+}
+
+@keyframes headerBtnClick {
+  0% {
+    transform: translateY(0);
+  }
+  100% {
+    transform: translateY(2px);
+  }
+}
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+/* Main */
+
+main {
+  border-bottom: 1px solid black;
+  margin-top: 40px;
+  padding-bottom: 10px;
+}
+
+section {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 39px;
+  line-height: 17px;
+}
+
+/*I know this is not the best way, I just wanted to practice with nth child :)*/
+section:nth-child(1) div:nth-child(2) {
+  margin-left: 14px;
+}
+
+section img {
+  width: 100%;
+}
+
+.section-div-3col:not(:first-child) {
+  padding-left: 18px;
+}
+
+/* Footer */
+
+footer {
+  padding-top: 38px;
+  display: flex;
+  flex-direction: column;
+  height: 235px;
+}
+
+footer span {
+  display: block;
+  margin-bottom: 2px;
+}
+
+.span-big-margin {
+  margin-top: 20px;
+}
+
+.copyright {
+  padding-top: 50px;
+  align-self: center;
+  font-size: 14px;
+}
+
+/***************800PX BREAK***********/
+
+@media (max-width: 800px) {
+  /*General*/
+  .container {
+    padding: 15px 0;
+    max-width: 760px;
+  }
+
+  /*Nav Area*/
+  .sticky-nav {
+    flex-direction: column-reverse;
+    clip-path: none;
+    margin: 0 auto;
+  }
+
+  .sticky-nav img {
+    padding: 4px 0 0 15px;
+  }
+
+  .sticky-nav a {
+    align-self: center;
+    font-size: 16px;
+  }
+
+  .sticky-nav nav ul {
+    margin: 40px auto;
+    padding: 0 35px 0 20px;
+    justify-content: space-between;
+  }
+
+  .nav-ul li:not(:first-child) {
+    margin-left: 0;
+  }
+
+  /*Header Area*/
+  .header-left {
+    padding: 0;
+  }
+
+  .header-right {
+    display: none;
+  }
+
+  .main-header {
+    justify-content: center;
+    margin: 35px 0 0 0;
+    padding: 0 0 0 16px;
+    height: 345px;
+  }
+  .main-header h1 {
+    letter-spacing: 2px;
+    font-size: 70px;
+    margin-bottom: 30px;
+  }
+
+  /*Main*/
+  main {
+    padding-left: 10px;
+    padding-bottom: 0;
+  }
+
+  main section {
+    margin-bottom: 35px;
+  }
+
+  main section div {
+    padding-right: 10px;
+  }
+
+  main section img {
+    margin-left: -8px;
+    width: 760px;
+    height: 160px;
+  }
+
+  main .section-div-3col {
+    margin-top: -6px;
+    padding-right: 25px;
+    line-height: 1.15;
+  }
+
+  /*Footer*/
+  footer {
+    padding-left: 10px;
+    padding-top: 35px;
+  }
+
+  footer span {
+    margin-bottom: 0;
+  }
+
+  .span-big-margin {
+    margin-top: 15px;
+  }
+
+  .copyright {
+    padding-top: 40px;
+    font-size: 16px;
+  }
+}
+
+/***************500PX BREAK***********/
+
+@media (max-width: 500px) {
+  /*General*/
```

Comment:
I would make your container at least 14px smaller than the breakpoint to account for the vertical scroll bar.

Reply:
Fixed